docs/block-elements.md

```diff
@@ -633,6 +633,24 @@ As same as `<Input type="hidden">`, `submit` prop in `<Modal>` must not define w
 
 It's exactly same as [`<Input>` component](#input-props), except `type` prop.
 
+## <a name="mrkdwn_component" id="mrkdwn_component"></a> [`<Mrkdwn>`: Markdown Composition Component](https://api.slack.com/reference/block-kit/composition-objects#text)
```

Comment:
Added documentation should be in a section of "Composition objects" (https://github.com/speee/jsx-slack/blob/master/docs/block-elements.md#composition-objects).

Please move it to below `<Confirm>` / right above `## Input components for modal`, and demote each headings one level (`##` -> `###`, `###` -> `####`).